### ISSUE TITLE
Graph edge labels not visible in dark theme

### ISSUE THREAD

**Issue by LarsKemmann** [microsoft/AzureStorageExplorer#650]
**Storage Explorer Version**: 1.4.1
**Platform/OS Version**: Windows
**Architecture**: ia32
**Regression From**: N/A

#### Steps to Reproduce: ####
1. Use dark theme
2. View a graph w/ edges that have labels

#### Expected Experience: ####
White-on-black labels for graph edges

#### Actual Experience: ####
Black-on-black, maybe? I can't really tell. There are no visible/legible labels on the graph edges, although I know that the labels have been populated (I can see the label values in the right-side properties panel)

**Comment by aliuy** [comment]
Azure Storage Explorer support for Azure Cosmos DB has been discontinued as of v1.24.0. We recommend users to use Azure Cosmos DB’s Data Explorer, which can be found in [Azure Portal](https://portal.azure.com/) or as a stand-alone version in http://cosmos.azure.com/ for the best Azure Cosmos DB experience. You can also find additional Azure Cosmos DB tooling here: [Azure Cosmos DB Tools](https://developer.azurecosmosdb.com/tools)
